include/grpcpp/impl/codegen/client_callback.h

```diff
@@ -174,6 +181,23 @@ class ClientBidiReactor {
   }
   void StartWritesDone() { stream_->WritesDone(); }
 
+  /// Holds are needed if this stream has operations that take place on it after
```

Comment:
Thanks for capturing all the important pieces in this comment!

Nit: Maybe we can add some newlines to break up the text such as before the "For example".